src/app/services/registry.service.ts

```diff
@@ -224,8 +223,9 @@ export class PluginRegistryBaseService {
             const responseData = await response.json() as T;
 
             if (isApiResponse(responseData)) {
+                const embedded = responseData?.embedded;
```

Comment:
to avoid similar errors, please remove `embedded` from the response here (see line 153/154) and update the signature of `cacheResults` to also include `embedded`. This way the surprise is no longer hidden in the `cacheResults` method...